TrackForce/src/test/java/com/revature/test/restAssured/UserResourceTest.java

```diff
@@ -33,17 +29,14 @@
  * @author Jesse
  * @since 06.18.06.16
  */
-/* Note by Seth L.: Every unhappy path test should have these test scenarios:
- *   1. Using an unsupported method
- *   2. Not using a token
- *   3. Using a bad token
- *   4. Using an unauthorized token (i.e. associate token)
- *   5. Bad data
+/*
```

Comment:
Each item in the numbered list should have its own line, like it was before the changes